api/nakadi-event-bus-api.yaml

```diff
@@ -613,6 +632,11 @@ paths:
             The flow id of the request, which is written into the logs and passed to called services. Helpful
             for operational troubleshooting and log analysis.
           type: string
+        - name: consumed_offset
```

Comment:
@dehora I'm not sure I understood what you mean by additive field. This property `unconsumed_events` is only present when `consumed_offset` is provided as a parameter in the query.